src/api/app/models/obs_factory/distribution.rb

```diff
@@ -209,13 +209,13 @@ def openqa_filter(project)
 
     def distribution_strategy_for_project(project)
       s = case project.name
-        when 'openSUSE:Factory' then DistributionStrategyFactory.new
-        when 'openSUSE:Factory:PowerPC' then DistributionStrategyFactoryPPC.new
-        when /^openSUSE:Leap:15\.*/ then DistributionStrategyOpenSUSELeap15.new
-        when /^SUSE:SLE-12-SP\d:GA/ then DistributionStrategySLE12SP1.new
-        when 'SUSE:SLE-15:GA' then DistributionStrategySLE15.new
-        when /^SUSE:SLE-15-SP\d:GA/ then DistributionStrategySLE15.new
-        when /^SUSE:SLE-12-SP.*CASP\d*/ then DistributionStrategyCasp.new
+          when 'openSUSE:Factory' then DistributionStrategyFactory.new
+          when 'openSUSE:Factory:PowerPC' then DistributionStrategyFactoryPPC.new
+          when /^openSUSE:Leap:15\.*/ then DistributionStrategyOpenSUSELeap15.new
+          when /^SUSE:SLE-12-SP\d:GA/ then DistributionStrategySLE12SP1.new
+          when 'SUSE:SLE-15:GA' then DistributionStrategySLE15.new
+          when /^SUSE:SLE-15-SP\d:GA/ then DistributionStrategySLE15.new
+          when /^SUSE:SLE-12-SP.*CASP\d*/ then DistributionStrategyCasp.new
         else raise UnknownDistribution
```

Comment:
`else` should be aligned with `when` too?

Reply:
Right. Aligned. Please re-review again.